### ISSUE TITLE
[A11y] Update honeypot module

### ISSUE THREAD

**Issue by lramarojaona** [openfed/openfed8#82]
There's an error triggered by A11y checker with the honeypot input which doesn't have a label.

This has been fixed in latest version: https://www.drupal.org/project/honeypot/releases/8.x-1.30

See the issue:
https://www.drupal.org/project/honeypot/issues/2944843

**Comment by rutiolma** [comment]
Updated.
